DatadogSessionReplay/Sources/DatadogSessionReplay/DatadogCoreIntegration/RUMContextReceiver.swift

```diff
@@ -15,6 +15,8 @@ internal struct RUMContext: Equatable {
     let sessionID: String
     /// Current RUM view ID - standard UUID string, lowecased.
     let viewID: String
+    /// Current RUM view server time offeset.
```

Comment:
Typo + would be easier to read if we know the unit
```suggestion
    /// Current RUM view server time offset (in milliseconds).
```